utilities/test_suite/rpp_test_suite_audio.h

```diff
@@ -245,7 +242,7 @@ void verify_output(Rpp32f *dstPtr, RpptDescPtr dstDescPtr, RpptImagePatchPtr dst
                 refVal = refPtrTemp[j];
                 outVal = dstPtrTemp[j];
                 bool invalidComparision = ((outVal == 0.0f) && (refVal != 0.0f));
-                if (!invalidComparision && abs(outVal - refVal) < 1e-20)
+                if (!invalidComparision && abs(outVal - refVal) < cutoff)
```

Comment:
@HazarathKumarM 
Are the QA tests passing in this PR?

If i am not wrong, MFB HIP is matching only till 5 decimal places

Reply:
Done